contributions/catalog/controllers/contribute.py

```diff
@@ -117,16 +118,17 @@ def create():
         response, s = post(json_contribution)
 
         if response:
-            if response:
-                if "name" in session:
-                    return render_template('contribute/submitted.html', user=session["name"],  token=session['oauth_token']['access_token'])
-                else:
-                    return render_template('contribute/submitted.html')
-            elif not response:
-                if "name" in session:
-                    return render_template('contribute/error.html', user=session["name"],  token=session['oauth_token']['access_token'], error_msg=s)
-                else:
-                    return render_template('contribute/error.html', error_msg=s)
+            if "name" in session:
+                return render_template('contribute/submitted.html', user=session["name"],  token=session['oauth_token']['access_token'])
+            else:
+                return render_template('contribute/submitted.html')
+        elif not response:
+            logging.error(s)
+            s = "There was a problem in POST process."
```

Comment:
```suggestion
            s = "Contribution submission failed. Please try again after some time!"
```

Reply:
@ywkim312, please see my suggestion. Otherwise, this looks ready to be merged. Thanks.

Reply:
changed.